src/__tests__/__snapshots__/Credentials-test.ts.snap

```diff
@@ -67,7 +67,7 @@ Object {
       "exp": 1485321134,
       "iat": 1485321133,
       "iss": "did:ethr:0xbc3ae59bc76f894822622cdef7a2018dbe353840",
-      "jwt": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NkstUiJ9.eyJpYXQiOjE0ODUzMjExMzMsInN1YiI6IjB4MTEyMjMzIiwiY2xhaW0iOnsiZW1haWwiOiJiaW5nYmFuZ2J1bmdAZW1haWwuY29tIn0sImV4cCI6MTQ4NTMyMTEzNCwiaXNzIjoiZGlkOmV0aHI6MHhiYzNhZTU5YmM3NmY4OTQ4MjI2MjJjZGVmN2EyMDE4ZGJlMzUzODQwIn0.y7AK2CZkCfMNM0k2WqAl7X7gg5ieBRVYa3mrI3Wfsq3JkH7mOd4J2XT_7eI5GIL2-tIVpnqMWVzCrvNwRRkzQQE",
```

Comment:
Updated these snapshots because the order of keys inside the JWT payload changed, even though the code used to create them did not.  I'm not sure, but this may possibly be an effect of switching `did-jwt` to typescript and/or using `microbundle` for building.  Let me know if anyone thinks this is an issue that needs to be investigated further.